pkg/checksdb/check.go

```diff
@@ -198,11 +199,11 @@ func (check *Check) SetResult(compliantObjects, nonCompliantObjects []*testhelpe
 
 	if len(nonCompliantObjects) > 0 {
 		check.Result = CheckResultFailed
-		check.FailureReason = resultObjectsStr
+		check.SkipReason = resultObjectsStr
```

Comment:
It might be better if the `SkipReason` field only contains a simple string explaining why the check has been skipped. The full context for the result objects is already available in the `Details` field.

If it the check has not been skipped then it should be left empty.

Reply:
fixed